test/integration/fixtures/publishing_api_payload.json

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
Related in email-alert-service:

https://github.com/alphagov/email-alert-service/blob/master/spec/integration/random_input_spec.rb#L10-L23